server/chat-commands/admin.ts

```diff
@@ -90,8 +90,11 @@ export const commands: ChatCommands = {
 			html += Chat.html`<div style="float:right;color:#888;font-size:8pt">[${user.name}]</div><div style="clear:both"></div>`;
 		}
 
-		html = `/uhtml${(cmd === 'changeuhtml' ? 'change' : '')} ${name},${html}`;
-		return html;
+		if (cmd === 'changeuhtml') {
+			room.addUserUhtmlchange(user, name, html);
```

Comment:
Our previous implementation did it by detecting `|uhtmlchange|`; the new one should probably do it by detecting `|/uhtmlchange`. I'm aware that that'd be a lot harder since you'd have to parse, so I guess I'm tentatively fine with this, but we should probably rip out the old implementation completely.

Assuming there was something wrong with the old implementation that makes the attributed version preferable; I'm going to need to hunt that down.

Reply:
`|uhtmlchange|` is being used by dynamic HTMLs from chat plugins (chat plugins don't really have an user). Those probably could be changed to use `|c:|` with `~` user however.